core/templates/components/rubrics-editor/rubrics-editor.component.ts

```diff
@@ -91,8 +91,30 @@ export class RubricsEditorComponent {
     this.explanationEditorIsOpen[difficulty][index] = true;
   }
 
-  isExplanationValid(difficulty: string, index: number): boolean {
-    return Boolean(this.editableExplanations[difficulty][index]);
+  isExplanationLengthValid(difficulty: string, idx: number): boolean {
+    return this.editableExplanations[difficulty][idx].length <= 300;
```

Comment:
Can we use a named constant instead of the number? It's harder to maintain hardcoded numbers otherwise.

Reply:
Done